Simple Incremental/Assets/Scripts/SpawnPoint.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/* Spawn point is used in conjunction with the TriggerPoint to spawn new objects
+ * When a trigger event is seen it will invoke this method to spawn objects
+ * via the configured manner.*/
+public class SpawnPoint : MonoBehaviour
+{
+    public GameObject objectPrefab; //Object that will be spawned
+    public int spawnCount; //Number of Objects to Spawn, 0 for infinite
+    public int spawnDelay; //Time in seconds to wait before spawning next object
+    public int triggerID; //ID should match value of the triggers triggerID
```

Comment:
Maybe use a GameObject or list of GameObjects here instead of an ID so that you can drag/drop the triggers that cause a spawner to start.